programmers/4일차/완주하지-못한-선수.js

```diff
@@ -0,0 +1,39 @@
+function solution(participant, completion) {
+    const objectCompletion = {};
+    
+    for (let  person of participant) {
+        objectCompletion[person] = objectCompletion[person] ? objectCompletion[person] + 1 : 1;
+        //이럴때 삼항 연산자 써본다 objectCompletion[person] 이 여러번 나오게 된다면 두번째 세번째 나오는 놈은 그 앞번에 나온녀석의 value값에 +1 해준값을 value로 주고
+        //조건이 거짓일때 즉, 한번만 나오는경우 1을 value에 넣어줌
+        console.log(objectCompletion)
+    }
+    for (let comPerson of completion) {
+        objectCompletion[comPerson] -= 1;
+        console.log(objectCompletion)
+    }
+    for (let key of participant) {
+        if (objectCompletion[key] == 1) {
```

Comment:
```suggestion
        if (objectCompletion[key]) {
```
이것도 전에 말씀드릴려고 했는데  지금 말씀드릴게요  ㅋㅋㅋㅋ    이렇게  고쳐도 풀 수 있어요 
이유는 자바스크립트에서는  값이 1인 상태를 true로  0인 상태를 false인 상태로 간주합니다  따라서  
= 1 을 뺄 수 있어요 지금 푸신 것도 절대 잘못된건 아니지만  이런 방법도 있다는 걸 알려드리고 싶었어요